src/components/Contacts/ContactDetails/ContactDetailsTab/Other/ContactDetailsOther.tsx

```diff
@@ -0,0 +1,86 @@
+import React from 'react';
+import { Box, Link, styled, Typography } from '@material-ui/core';
+import { useTranslation } from 'react-i18next';
+import { ContactDetailsTabQuery } from '../ContactDetailsTab.generated';
+
+const ContactOtherContainer = styled(Box)(({ theme }) => ({
+  marginLeft: theme.spacing(5),
+  margin: theme.spacing(1),
```

Comment:
You could maybe consolidate this down do:
```suggestion
  margin: theme.spacing(1, 1, 1, 5)
```
but I'm not sure if that's really any better...